pkg/serverless/trace/inferredspan/span_enrichment.go

```diff
@@ -288,17 +288,29 @@ func (inferredSpan *InferredSpan) EnrichInferredSpanWithSQSEvent(eventPayload ev
 // payload to enrich the current inferred span. It applies a
 // specific set of data to the span expected from an EventBridge event.
 func (inferredSpan *InferredSpan) EnrichInferredSpanWithEventBridgeEvent(eventPayload events.EventBridgeEvent) {
```

Comment:
Am I understanding this correctly? This `EnrichInferredSpanWithEventBridgeEvent` method existed in the extension already, but it just wasn't being used anywhere?

Reply:
Yes that's correct. It was supposed to be called [here](https://github.com/DataDog/datadog-agent/blob/3b96ef1a258e773bab62a1d0b355bb6186a9e51b/pkg/serverless/invocationlifecycle/init.go#L109-L111)